### ISSUE TITLE
Not compatible with v>1.5.0

### ISSUE THREAD

**Issue by nohorbee** [mikalhart/Flash#6]
I found this after a while without working on a project that included this library.
There are 2 pull requests that actually solve the problem. I forked from them and included as submodule of my project, and everything goes well (that's a good test).

Any plan of accepting those Pull Requests?

**Comment by mvhconsult** [comment]
This indeed would be nice. Please grant the pull request.

**Comment by nohorbee** [comment]
Hi, I also patched this to be compatible with v>1.6.5. Please check the Pull Requests I made.

**Comment by mvhconsult** [comment]
Maybe someone else could take this project over and also add it to the official library? Three years without commits looks like the project is dead here.

**Comment by nohorbee** [comment]
@mvhconsult. I could do it. But I would also continue to do pull requests to the forked repo (mine is the 3rd level and Te 2nd level merged my PR yesterday). @schinken, do you feel comfortable having the official repo? I could help if you like me to.

**Comment by schinken** [comment]
Yes, I can have the "official" repo. I can also add you to the repo members if you like.

**Comment by mvhconsult** [comment]
I don't mind adding some to the repo (docs, examples), but I vote for the most practical repo now, which seems to be @schinken.

**Comment by nohorbee** [comment]
I agree. @schinken, go ahead ☺

**Comment by synfinatic** [comment]
awesome. go @schinken !

**Comment by mvhconsult** [comment]
For all interested: we are continuing at https://github.com/schinken/Flash
